src/main/webapp/WEB-INF/tags/instructor/course/archivedCoursesTable.tag

```diff
@@ -0,0 +1,33 @@
+<%@ tag description="instructorCourse - Course table" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib prefix="fn" uri="http://java.sun.com/jsp/jstl/functions" %>
+
+<h2 class="text-muted">Archived courses</h2>
+<table class="table table-bordered table-striped">
+    <thead>
+        <tr class="fill-default">
+            <th onclick="toggleSort(this,1);" id="button_sortid" class="button-sort-none">
+                Course ID<span class="icon-sort unsorted"></span>
+            </th>
+            <th onclick="toggleSort(this,2);" id="button_sortid" class="button-sort-none">
+                Course Name<span class="icon-sort unsorted"></span>
+            </th>
+            <th class="align-center no-print">Action(s)</th>
+        </tr>
+    </thead>
+    <c:forEach items="${data.archivedCourses.rows}" var="archivedCourse" varStatus="i">
```

Comment:
Please update this to require the object as an attribute instead of getting it from `data`.